Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -85,36 +75,61 @@ final class PledgeDescriptionCell: UITableViewCell, ValueCell {
 
     _ = self.learnMoreTextView
       |> learnMoreTextViewStyle
+
+    _ = self.rewardCardContainerView
+      |> rewardCardContainerViewStyle
+
+    _ = self.rewardCardContainerMaskView
+      |> rewardCardContainerMaskViewStyle
+  }
+
+  override func layoutSubviews() {
+    super.layoutSubviews()
+
+    self.sizeAndTransformRewardCardView()
   }
 
   private func configureSubviews() {
     _ = (self.rootStackView, self.contentView)
       |> ksr_addSubviewToParent()
       |> ksr_constrainViewToEdgesInParent()
 
-    _ = ([self.containerImageView], self.rootStackView)
+    _ = ([self.rewardCardContainerMaskView], self.rootStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    _ = (self.pledgeImageView, self.containerImageView)
+    _ = (self.rewardCardView, self.rewardCardContainerView)
       |> ksr_addSubviewToParent()
-      |> ksr_constrainViewToEdgesInParent()
+      |> ksr_constrainViewToMarginsInParent()
 
-    self.configureStackView()
+    _ = (self.rewardCardContainerView, self.rewardCardContainerMaskView)
+      |> ksr_addSubviewToParent()
+
+    self.rewardCardContainerMaskViewWidthConstraint = self.rewardCardContainerMaskView.widthAnchor
+      .constraint(equalToConstant: 0)
+    self.rewardCardContainerMaskViewHeightConstraint = self.rewardCardContainerMaskView.heightAnchor
+      .constraint(equalToConstant: 0)
+
+    self.rewardCardContainerMaskViewHeightConstraint?.priority = .defaultLow
+
+    let rewardConstainerConstraints = [
+      self.rewardCardContainerMaskViewWidthConstraint,
+      self.rewardCardContainerMaskViewHeightConstraint
+    ]
+    .compact()
 
     NSLayoutConstraint.activate([
-      self.containerImageView.widthAnchor.constraint(equalToConstant: Layout.ImageView.width),
-      self.containerImageView.heightAnchor.constraint(equalToConstant: Layout.ImageView.height),
-      self.pledgeImageView.centerXAnchor.constraint(equalTo: self.containerImageView.centerXAnchor)
-    ])
+      self.rewardCardView.widthAnchor.constraint(equalToConstant: RewardCardView.cardWidth),
+      self.rewardCardContainerView.leftAnchor.constraint(
```

Comment:
`rewardCardContainerView` only has it's `top` and `left` anchors pinned to `rewardCardContainerMaskView`, this is so that it can grow as large as it needs to and then be transformed into the thumbnail size.